config/packages/kimai.yaml

```diff
@@ -253,6 +253,12 @@ kimai:
             date_type: 'dd.MM.yyyy'
             date: 'd.m.Y'
             date_time: 'd.m. H:i'
+        el:  
+            date_time_type: 'dd.MM.yyyy HH:mm'
+            date_type: 'dd.MM.yyyy'
+            date: 'd.m.Y'
+            date_time: 'd.m. H:i'
+            duration: '%%h:%%m h'
```

Comment:
```suggestion
```

Reply:
Well i am OK with that usually we use the same format as the rest of Europe,  with translated markings though.

Reply:
yeah, you only need lines that are different from english - and English already uses `%%h:%%m h`